apps/api/app/agents/tools/core/retrieval.py

```
@@ -9,68 +9,271 @@
+    # Search subagents namespace
+    if include_subagents:
+        logger.info("Adding search for subagents namespace")
+        search_tasks.append(store.asearch(("subagents",), query=query, limit=15))
+        search_tasks.append(search_public_integrations(query=query, limit=15))
+
+    return search_tasks
+
+
+def _process_public_integration_result(
+    result: List[dict[str, Any]],
+    task_idx: int,
```

Ruff (0.14.14)

Unused function argument: `task_idx`

(ARG001)